client/modules/IDE/components/Editor.jsx

```diff
@@ -346,25 +346,23 @@ class Editor extends React.Component {
         }
       );
       this._cm.doc.setValue(formatted);
-      return cursorOffset;
+      if (cursorOffset) {
```

Comment:
You also can remove the `if` statement now as the `cursorOffset` is always non zero.
If we want to be specific we can do it by `if(cursorOffset==this._cm.doc.indexFromPos(this._cm.doc.getCursor()))`